ArduCopter/mode_auto.cpp

```diff
@@ -453,15 +453,15 @@ bool ModeAuto::wp_start(const Location& dest_loc)
 {
     // init wpnav and set origin if transitioning from takeoff
     if (!wp_nav->is_active()) {
-        Vector3p stopping_point_neu_m;
+        Vector3p stopping_point_ned_m;
         if (_mode == SubMode::TAKEOFF) {
-            Vector3p takeoff_complete_pos_neu_m;
-            if (auto_takeoff.get_completion_pos_neu_m(takeoff_complete_pos_neu_m)) {
-                stopping_point_neu_m = takeoff_complete_pos_neu_m;
+            Vector3p takeoff_complete_pos_ned_m;
+            if (auto_takeoff.get_completion_pos_ned_m(takeoff_complete_pos_ned_m)) {
+                stopping_point_ned_m = takeoff_complete_pos_ned_m;
             }
         }
         float des_speed_xy_ms = is_positive(desired_speed_override_ms.xy) ? desired_speed_override_ms.xy : 0;
-        wp_nav->wp_and_spline_init_m(des_speed_xy_ms, stopping_point_neu_m);
+        wp_nav->wp_and_spline_init_m(des_speed_xy_ms, stopping_point_ned_m);
```

Comment:
not renaming WPNav's wp_and_spline_init_m method means we will need to be very careful that we find all instances where this is used or the caller could be surprised that the vehicle climbs or descends.  I think we've renamed it already as part of the move to meters so external developers will hopefully bump into the issue when they rebase.